transmart-server/grails-app/services/org/transmart/server/subscription/QuerySetSubscriptionMailService.groovy

```diff
@@ -3,6 +3,7 @@ package org.transmart.server.subscription
 import grails.plugins.mail.MailService
 import groovy.util.logging.Slf4j
```

Comment:
Sorry for noticing this only now. Why do we have this logic in `transmart-server`? Saying that I understand that neither `transmart-core-db`, nor `transmart-rest-api` are proper places. What about having a new module e.g. `transmart-mail`?